dev/env/scripts/lib.sh

```diff
@@ -86,6 +86,10 @@ init() {
         source "$env_file"
     done
 
+    if [[ -z "${CENTRAL_IDP_CLIENT_SECRET}" ]]; then
```

Comment:
```suggestion
    if [[ -z "${CENTRAL_IDP_CLIENT_SECRET:-}" ]]; then
```

otherwise it would abort due to accessing an uninitialized variable. See:

```
$ bash -c 'source dev/env/scripts/lib.sh; init; if [[ -z "${NONEXISTANT}" ]]; then echo "is empty"; fi'
bash: NONEXISTANT: unbound variable
```